src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0">
```

Comment:
```suggestion
      <div class="stars stars--0">
```

Reply:
remember one more time how BEM modifiers work

Reply:
I don't understand what needs to be fixed, I haven't passed the BEM module yet and the tests are looking for stars--N

Reply:
need to add the 'stars' class?